roles/prometheusexporter/templates/grafanadashboard-es.yml.j2

```diff
@@ -2012,54 +2012,775 @@ spec:
             "align": false,
             "alignLevel": null
           }
-        }
-      ],
-      "refresh": "30s",
-      "schemaVersion": 18,
-      "style": "dark",
-      "tags": [
-        "prometheus-exporter",
-        "es"
-      ],
-      "templating": {
-        "list": [
-          {
-            "hide": 0,
-            "includeAll": false,
-            "label": null,
-            "multi": false,
-            "name": "datasource",
-            "options": [],
-            "query": "prometheus",
-            "refresh": 1,
-            "regex": "",
-            "skipUrlSync": false,
-            "type": "datasource"
+        },
+        {
+          "collapsed": false,
+          "datasource": null,
+          "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 61
           },
-          {
-            "allValue": null,
-            "current": {
-              "text": "{{ meta.namespace }}",
-              "value": "{{ meta.namespace }}"
+          "id": 103,
+          "panels": [],
+          "title": "System",
+          "type": "row"
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": null,
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 12,
+            "x": 0,
+            "y": 62
+          },
+          "hiddenSeries": false,
+          "id": 101,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "dataLinks": []
+          },
+          "percentage": false,
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "(elasticsearch_filesystem_data_size_bytes{cluster=~'.*:pro-base-es-logs', es_master_node=\"true\"} - elasticsearch_filesystem_data_free_bytes{cluster=~'.*:pro-base-es-logs', es_master_node=\"true\"}) / elasticsearch_filesystem_data_size_bytes{cluster=~'.*:pro-base-es-logs', es_master_node=\"true\"} * 100",
+              "legendFormat": "{{ '{{' }} instance {{ '}}' }}",
```

Comment:
This apply to other expressions also.

I have checked that `instance` label (used as legend) is not very descriptive, its pod-ip:pod-port (example `10.130.2.94:9114`), maybe we should add something more descriptive, you can check more labels at https://console-openshift-console.apps.pro-base-ocp4.mgmt.3sca.net/monitoring/query-browser?query0=elasticsearch_os_mem_used_bytes

Maybe we should use another label more descriptive to show as legend?